content/powershell/concepts/functions/functions.md

```diff
@@ -1,11 +1,121 @@
 ---
 Title: 'Functions'
-Description: ''
+Description: 'A function is a set of instructions that are executed together whenever invoked via its name. Parameters and importing modules can further enhance the usage of functions.'
 Subjects:
+  - 'Code Foundations'
   - 'Computer Science'
+  - 'Bash/Shell'
 Tags:
+  - 'Arguments'
   - 'Functions'
+  - 'Parameters'
 CatalogContent:
   - 'learn-powershell'
   - 'paths/computer-science'
 ---
+
+A **function** is a set of statements that are executed when the function name is called. Functions allow programmers to break down a problem into smaller chunks and help define a particular task. A few of the major reasons for using functions are readability and reusability.
```

Comment:
```suggestion
A **function** is a set of statements that are executed when the function name is called. Functions allow programmers to break down a problem into smaller chunks and help define a particular task. A couple of the major reasons for using functions are readability and reusability.
```

Reply:
Fixed in 23f54309efceaf758722ab60f2a5dde33e0f9e65